tests/Fixtures/Integration/priority/no_whitespace_in_blank_line,no_blank_lines_after_phpdoc.test

```diff
@@ -1,22 +0,0 @@
---TEST--
```

Comment:
please do not remove tests.
ever

Reply:
Oh, yeah, I haven't thought about moving it from `priority` to `misc`. Next time I will remember.